jupyterbook/content/ioos_installation_conda.md

```diff
@@ -42,22 +42,18 @@ elif [[ $(uname) == "Linux" ]]; then
   url=https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
 fi
 curl $url -o miniconda.sh
-bash miniconda.sh -b
+sh miniconda.sh
 export PATH=$HOME/miniconda3/bin:$PATH
 ```
 
-We also recommend to add the following line to your `~/.bashrc` file to make Miniconda the Python found first than the system Python:
+and use all the default options, expect for the license agreement where you must actively change it to `yes`.
```

Comment:
expect -> except

Reply:
Spellcheck is worthless when your typo exist :-)